File: a003(divination)/C++/divination.cpp
#include <iostream>

int main(void) {
  unsigned int month, date;
  std::cin >> month >> date;
  unsigned int s = (month*2 + date) % 3;
  switch (s) {
    case 0:
      std::cout << "普通" << '\n';
      break;
    case 1:
      std::cout << "吉" << '\n';
      break;
    case 2:
      std::cout << "大吉" << '\n';
      break;
  }
  return 0;
}

File: a003(divination)/Java/Divination.java
import java.util.Scanner;


public class Divination {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int month = in.nextInt();
        int date  = in.nextInt();
        int s = (month * 2 + date) % 3;
        switch (s) {
            case 0:
                System.out.println("普通");
                break;
            case 1:
                System.out.println("吉");
                break;
            case 2:
                System.out.println("大吉");
                break;
            default:
                break;
        }
    }
}
